Dockerfile

```diff
@@ -10,4 +10,5 @@ RUN apk add --update \
 
 RUN npm i -g
 
-ENTRYPOINT "npm-groovy-lint"
+ENTRYPOINT ["npm-groovy-lint"]
+CMD ["--help"]
```

Comment:
This differs from running `npm-groovy-lint`. 
Should the default behaviour be changed?

Reply:
You're right.  In order to avoid breaking the default behavior and potentially angering hordes of users who don't specify their docker tags, we can revert this.  In my testing we can still do `docker run -u "$(id -u):$(id -g)" -w=/tmp -v "$PWD":/tmp nvuillam/npm-groovy-lint --help` to achieve the same result as long as the exec form is used.